custom_components/ha_creality_ws/camera.py

```
@@ -723,6 +756,87 @@ async def _ensure_stream_configured(self) -> None:
+        if value.startswith("webrtc:"):
+            value = value[len("webrtc:"):]
+        return value.split("#", 1)[0]
+
+    def _go2rtc_source_url(self) -> str:
+        """Return the go2rtc source URL for this printer's WebRTC protocol."""
+        source = f"webrtc:{self._upstream_signaling_url}"
+        try:
+            if ModelDetection(getattr(self.coordinator, "data", {})).is_k2_family:
+                return f"{source}#format=creality"
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)